crates/sui-core/src/unit_tests/transaction_tests.rs

```diff
@@ -453,15 +451,15 @@ async fn test_zklogin_transfer_with_bad_ephemeral_sig() {
     .await;
 }
 
-fn zklogin_key_pair_and_inputs() -> Vec<(Ed25519KeyPair, ZkLoginInputs)> {
-    let key1 = Ed25519KeyPair::generate(&mut StdRng::from_seed([1; 32]));
-    let key2 = Ed25519KeyPair::generate(&mut StdRng::from_seed([2; 32]));
+// fn zklogin_key_pair_and_inputs() -> Vec<(Ed25519KeyPair, ZkLoginInputs)> {
```

Comment:
Is this still WIP?

Reply:
its ready for review! just pushed some lints and fixes